src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer start working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("bulldozer stop working");

```

Comment:
The message in the 'stopWork' method should be in English and start with a capital letter to maintain consistency and professionalism. Additionally, the message should be more informative. For example: 'Bulldozer has stopped its work.'